aten/src/ATen/native/native_functions.yaml

```diff
@@ -1138,6 +1138,10 @@
     SparseCUDA: bmm_out_sparse_cuda
     SparseCsrCUDA: bmm_out_sparse_csr_cuda
 
+- func: _NestedTensor_GeneralizedBMM(Tensor self, Tensor mat2) -> Tensor
+  dispatch:
+    NestedTensorCPU, NestedTensorCUDA: _NestedTensor_GeneralizedBMM
```

Comment:
This is not needed, we can do the exact same thing as what was done for linear: https://github.com/pytorch/pytorch/pull/80397